docs/navigation-parameters-reference.md

```diff
@@ -1,295 +1,309 @@
-  -   **title** modal title. By default, it is the node label. If there is no label, it is left empty
-  -   **size** **(`"l"` \| `"m"` \| `"s"`)** size of the modal (optional, default `"l"`)
-- **onNodeActivation** is an optional function executed when a request to navigate to the node occurs. As an input parameter, the function receives the node object as described in the configuration. This function can return results synchronously or asynchronously. If the function returns boolean `false`, the navigation is not triggered, otherwise, navigation renders as usual.
-- **clientPermissions.changeCurrentLocale** current locale can be changed from client using the corresponding API if this is set to `true`
-- **tabNav** is a boolean which renders the children of the node as a horizontal navigation bar. Sub-children are not supported. When you categorize nodes you will get a drop-down menu in the horizontal navigation.
+
+### category
+- **type**: string or object
+- **description**: defines a group of views separated with a headline and an icon. You should define at least one node in a group should as an Object with **label** and **icon** attributes. For all other nodes, you can set **category** as a string with the `label` value.
+- **attributes**:
+  - **label** is a string that represents the title of the category
+  - **icon** is the name of an icon, without the `sap-icon--` prefix. Its source may be [OpenUI](https://openui5.hana.ondemand.com/1.40.10/iconExplorer.html) or a custom link (relative or absolute) to an image. The icon is displayed next to the node label in the side navigation or instead of the label in the top navigation. In case you accidentally define different icons in a category group, only the first one is used.
+  - **collapsible** if set to `true`, category items are hidden at first. To expand them, click the main category node.
+  - **testId** is a string where you can define your own custom `testId`. If nothing is specified, it is the node's label written as one word and lower case (e.g. `label`).
+
+### openNodeInModal
+- **type**: boolean or object
+- **description**:  configures the settings of a view which opens in a modal. You can set the openNodeInModal property to true to use the default modal title and size, or you can specify them using these attributes:
+- **attributes**:
+  - **title** is the modal title. By default, it is the node label. If there is no label, it is left empty.
+  - **size** **(`"l"` \| `"m"` \| `"s"`)** specifies the size of the modal. The default size is **`"l"`**, which means 80%. You can also use **`"m"`** (60%) and **`"s"`** (40%) to set the modal size.
```

Comment:
Are the values here escaped on purpose? If not, you can change the formatting to regular one. ** for property, ` for value.